Physical_Engine/Query/Volume.cs

```diff
@@ -0,0 +1,67 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
+using BH.oM.Physical.Elements;
+using BH.oM.Geometry.ShapeProfiles;
+using BH.oM.Geometry;
+using BH.oM.Physical.FramingProperties;
+using BH.oM.Physical.Materials;
+using BH.Engine.Geometry;
+using BH.Engine.Reflection;
+
+namespace BH.Engine.Physical
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+        [Description("Returns the volume of a column")]
```

Comment:
Would expand the description here a bit to say what the method is doing (and thereby not doing). What I mean by this is stating that this is simply taking the length of the centre line times the area of the profile, nothing more, nothing less. This is so that it is clear to everyone that this will not account for cutouts, connections, plates, bolts etc etc.

Reply:
Yep - agreed on this @IsakNaslundBh 
Might we go as far as raising a friendly warning?

Reply:
Or even call the method - `LinearExtrudedVolume` or something similar?